src/Serilog/Context/LogContext.cs

```diff
@@ -12,9 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+
 #if LOGCONTEXT
```

Comment:
Sorry if this is a duplicate, I may have already asked - but are there any platforms on which `LogContext` is unsupported now? (I.e. is `AsyncLocal` on 5.1?)